Userland/Libraries/LibC/time.cpp

```diff
@@ -204,8 +210,8 @@ char* asctime(const struct tm* tm)
 
 char* asctime_r(const struct tm* tm, char* buffer)
 {
-    // Spec states buffer must be at least 26 bytes.
-    constexpr size_t assumed_len = 26;
+    // Spec states buffer must be at least 33 bytes.
+    constexpr size_t assumed_len = 33;
```

Comment:
This blatantly violates the spec.

Reply:
I was having problem with this since years now are taking too much space. For larger time_t, we have to append 10 numbers and thus, 26 bytes are not sufficient. I will try to code an alternative based on the passed buffer.

Reply:
There is no way to tell the buffer size just from the pointer. We should only assume 26 bytes of space to be present (25 characters and a null byte), period. Everything else is an error and should set ´errno` and return a `nullptr` accordingly.